packages/fxa-auth-server/fxa-oauth-server/docs/api.md

```diff
@@ -330,16 +330,18 @@ back to the client. This code will be traded for a token at the
 - `client_id`: The id returned from client registration.
 - `assertion`: A FxA assertion for the signed-in user.
 - `state`: A value that will be returned to the client as-is upon redirection, so that clients can verify the redirect is authentic.
-- `response_type`: Optional. If supplied, must be either `code` or `token`. `code` is the default. `token` means the implicit grant is desired, and requires that the client have special permission to do so.
-  - **Note: new implementations should not use `response_type=token`; instead use `grant_type=fxa-credentials` at the [token][] endpoint.**
-- `ttl`: Optional if `response_type=token`, forbidden if `response_type=code`. Indicates the requested lifespan in seconds for the implicit grant token. The value is subject to an internal maximum limit, so clients must check the `expires_in` result property for the actual TTL.
-- `redirect_uri`: Optional. If supplied, a string URL of where to redirect afterwards. Must match URL from registration.
-- `scope`: Optional. A string-separated list of scopes that the user has authorized. This could be pruned by the user at the confirmation dialog.
-- `access_type`: Optional. A value of `offline` will generate a refresh token along with the access token.
+- `access_type`: (optional) A value of `offline` will generate a refresh token along with the access token.
+- `acr_values`: (optional) A string-separated list of acr values that the token should have a claim for. Specifying `AAL2` will require the token to have an authentication assurance level >= 2 which corresponds to requiring 2FA.
 - `code_challenge_method`: Required if using [PKCE](pkce.md). Must be `S256`, no other value is accepted.
 - `code_challenge`: Required if using [PKCE](pkce.md). A minimum length of 43 characters and a maximum length of 128 characters string, encoded as `BASE64URL`.
-- `keys_jwe`: Optional. A JWE bundle to be returned to the client when it redeems the authorization code.
-- `acr_values`: Optional. A string-separated list of acr values that the token should have a claim for. Specifying `AAL2` will require the token to have an authentication assurance level >= 2 which corresponds to requiring 2FA.
+- `keys_jwe`: (optional) A JWE bundle to be returned to the client when it redeems the authorization code.
+- `ppid_seed`: (optional) Seed used in `sub` claim generation of JWT access tokens/ID tokens for clients with Pseudonymous Pairwise Identifiers (PPID) enabled.
```

Comment:
I haven't removed this yet, but have realized I put `ppid_seed` in the `GET /authorization` section instead of `POST /authorization`. Moving for now to think about whether removing support is the right thing to do.

Reply:
Removed support, it makes no sense to add it to the POST /authorization endpoint if we don't surface a way for RPs to provide a ppid_seed on the GET /authorization endpoint and have it propagated via the content server.